src/luarocks/core/cfg.lua

```diff
@@ -416,6 +422,29 @@ local function make_defaults(lua_version, target_cpu, platforms, home)
          lib = { "lib?.so", "?.dll", "lib?.dll" },
          include = { "?.h" }
       }
+      if platforms.mingw then
+         -- MSYS2 can build Windows programs that depend on
+         -- msys-2.0.dll (based on Cygwin) but MSYS2 is also designed
+         -- for building native Windows programs by MinGW. These
+         -- programs don't depend on msys-2.0.dll.
+         defaults.makefile = "Makefile"
+         defaults.cmake_generator = "MSYS Makefiles"
+         defaults.local_cache = home.."/.cache/luarocks"
+         defaults.variables.PWD = "cd"
```

Comment:
Use Windows' `cd` (not `cd` on bash) to get Windows style path.
No argument Windows' `cd` returns the current folder.